### PR TITLE
Tidy up v2 layouts

### PR DESCRIPTION
**Pull request by sean-brydon**
Tidies up layouts. 

Layouts that have been used in features have been also moved to the respective feature folders.
Anything else that isnt being reused have been moved into web/ui/components. As they are not being shared across the app.

### CODE REVIEW

**Comment by PeerRich** [comment]
great stuff!

File: packages/features/settings/layouts/SettingsLayout.tsx [hunk 1]
```diff
@@ -164,11 +167,11 @@ const SettingsSidebarContainer = ({
   return (
     <nav
       className={classNames(
-        "no-scrollbar fixed top-0 bottom-0 z-10 flex max-h-screen w-56 flex-col space-y-1 overflow-x-hidden overflow-y-scroll bg-gray-50 px-2 pb-3 transition-transform ltr:left-0 rtl:right-0 lg:sticky lg:flex",
+        "no-scrollbar fixed left-0 top-0 z-10 flex max-h-screen w-56 flex-col space-y-1 overflow-x-hidden overflow-y-scroll bg-gray-50 px-2 pb-3 transition-transform lg:sticky lg:flex",
         className,
         navigationIsOpenedOnMobile
           ? "translate-x-0 opacity-100"
-          : "opacity-0 ltr:-translate-x-full rtl:translate-x-full lg:translate-x-0 lg:opacity-100"
+          : "-translate-x-full opacity-0 lg:translate-x-0 lg:opacity-100"
```
Comment (JeroenReumkens) [inline packages/features/settings/layouts/SettingsLayout.tsx:174]: Did you have a merge conflict here? I think Peer's recent changes for RTL support got lost here.

**Review by JeroenReumkens — APPROVED** [review]

**Comment by JeroenReumkens** [comment]
@sean-brydon  Looks good, as soon as build and merge conflicts are fixed we're good to go imo.

File: packages/features/settings/layouts/SettingsLayout.tsx (hunk 1 above)
Comment (PeerRich) [reply packages/features/settings/layouts/SettingsLayout.tsx:174]: i had to undo my release. ltr rtl wasnt working

Comment (sean-brydon) [reply packages/features/settings/layouts/SettingsLayout.tsx:174]: Oh - this will need reverting

**Pull request merged**